Multisig/Data/Services/Networking Utils/HTTPClient.swift

```diff
@@ -81,6 +81,8 @@ public class HTTPClient {
     }
 
     private func send(_ request: URLRequest) -> URLDataTaskResult {
```

Comment:
this should break the `SafeRelayServiceIntegrationTests` - please update from master and run locally

Reply:
fixed integration tests